tests/Console/ConsoleParserTest.php

```diff
@@ -144,4 +145,12 @@ public function testOptionDefaultValue()
         $results = Parser::parse('command:name {--option=default : The option description.}');
         $this->assertSame('default', $results[2][0]->getDefault());
     }
+
+    public function testNameException()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Unable to determine command name from signature.');
+
+        Parser::parse(" \t\n\r\x0B\f");
```

Comment:
Wouldnt a better test be `Parser::parse("")` and prove the new code will still throw an `InvalidArgumentException`?

Reply:
This test asserts that the spaces are deleted in regex and if the string consists of only spaces, the exception will also appear